File: Array/Javascript/sum_of_array.js
const sumOfArray = (arr) => {
    let sumOfArray = 0
    for (let i = 0; i < arr.length; i++) {
        sumOfArray += arr[i]
    }
    return sumOfArray
}

const array = [1, 2, 3, 4, 5, 7, 8]
console.log(sumOfArray(array))

File: Array/Go/sum_of_array.go
package main

import "fmt"

func sumOfArray(arr []int) int {
	sum := 0
	for i := 1; i <= len(arr); i++ {
		sum += i
	}
	return sum
}

func main() {
	fmt.Println(sumOfArray([]int{1, 2, 3, 4, 5, 4, 54, 5, 4, 5, 4, 5, 4, 5, 4}))
}
